lib/common/tests/scores/char2score_test.c

```diff
@@ -56,8 +56,10 @@ special_values(void **state)
 static void
 outside_limits(void **state)
 {
-    assert_int_equal(char2score(B(CRM_SCORE_INFINITY) "00"), CRM_SCORE_INFINITY);
-    assert_int_equal(char2score("-" B(CRM_SCORE_INFINITY) "00"), -CRM_SCORE_INFINITY);
+    assert_int_equal(char2score(B(CRM_SCORE_INFINITY) "00"),
```

Comment:
These `CRM_SCORE_INFINITY` occurrences in `char2score()` didn't get updated to `PCMK_SCORE_INFINITY`, which is probably what's causing the unit test to fail.

Reply:
whoops, thanks